modules/gdscript/doc_classes/@GDScript.xml

```diff
@@ -620,10 +620,10 @@
 			<param index="3" name="transfer_channel" type="int" default="0" />
 			<description>
 				Mark the following method for remote procedure calls. See [url=$DOCS_URL/tutorials/networking/high_level_multiplayer.html]High-level multiplayer[/url].
-				The accepted values:
-				- for [param mode] are [code]"any_peer"[/code] or [code]"authority"[/code];
-				- for [param sync] are [code]"call_remote"[/code] or [code]"call_local"[/code];
-				- and for [param transfer_mode] are [code]"unreliable"[/code], [code]"unreliable_ordered"[/code], or [code]"reliable"[/code].
+				If [param mode] is set as [code]"authority"[/code], only the authority peer can call this RPC function. Otherwise, any peer can call it and this parameter should be set to [code]"any_peer"[/code]. When configuring functions as RPC with [method Node.rpc_config], each of these modes respectively corresponds to the [constant MultiplayerAPI.RPC_MODE_AUTHORITY] and [constant MultiplayerAPI.RPC_MODE_ANY_PEER] rpc modes. See [enum MultiplayerAPI.RPCMode]. If a peer that is not the authority tries to call a function that is only allowed for the authority, this will print an error on both the local and the remote peers, and the function will not be executed.
```

Comment:
I think this is a bit misleading, because it says "otherwise, any peer can call it and this parameter [...]" but the default (which happens if you don't specify neither `authority` nor `any_peer` is == authority).
So the phrase should be swapped (if is set to any_peer, otherwise).

Reply:
> this will print an error on both the local and the remote peers, and the function will not be executed.

Similarly, this might be a bit confusing, as you can have cases where no error is reported on the sender.
That's the case where the client is misconfigured (i.e. it thinks it can send it, e.g. due to having a different authority setting from the one on the server).